pkg/memory-monitor/memory-monitor-ctr-init.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Do we need this starter and daemonization if you can run the whole monitor as a tool from Dockerfile directly?

Reply:
We discussed it with @rucoder, and it looks like it's still necessary. The problem is that the tool is demonized, which means it goes to the background and returns control to the caller. But our boot system (what's the proper name? LinuxKit? containerd?) does not expect a service to return control; that's why we need a wrapper with an infinite loop.

Reply:
There is even a comment about it in the script =)

Reply:
but you can just remove this, can't you? - https://github.com/lf-edge/eve/pull/3984/files#diff-f17e11f9dca74dfea2d7e3b33156609477e2745002fa9f7fc072c0490b746100R85

Reply:
That what I'm saying, why you need to daemonize the tool if you wait for it?

Reply:
To clarify, I've added a section about the standalone run into the documentation.

Reply:
Sure thing, keep it as you wish, but initial decision should not stop you from implementing the simple flag 
`  -d            Don't run as daemon and log to stderr'` (copied from chronyd).

Reply:
That makes sense. @rucoder also insisted on such a flag. I will add the flag then and retest how the tool behaves in a foreground version. At least I had some problems with the daemonized behaviour when I implemented it (logging, basically, but I can forget something else).

Reply:
Done